common/src/main/scala/uk/ac/wellcome/models/display/DisplayAgent.scala

```diff
@@ -1,4 +1,4 @@
-package uk.ac.wellcome.platform.api.models
+package uk.ac.wellcome.models.display
```

Comment:
👍 good use of namespacing.

Reply:
This also means we can drop `Display` from the name of the classes.

Reply:
Having thought about it, I think I’ll keep the Display prefixes. Yes it’s double-namespacing, but there are contexts where it makes a difference – e.g. in the API, where at various points you’re dealing with internal/display Works.

For human-readability, I think I’ll leave it (else I’d change all the imports to `import uk.ac.wellcome.models.display.{Work => DisplayWork}` anyway!)